src/components/Tool/FixCommitmentInfo/FixCommitmentInfo.test.tsx

```diff
@@ -0,0 +1,202 @@
+import React from 'react';
+import { ThemeProvider } from '@mui/material/styles';
+import userEvent from '@testing-library/user-event';
+import { ErgonoMockShape } from 'graphql-ergonomock';
+import { SnackbarProvider } from 'notistack';
+import { VirtuosoMockContext } from 'react-virtuoso';
+import TestRouter from '__tests__/util/TestRouter';
+import TestWrapper from '__tests__/util/TestWrapper';
+import { GqlMockedProvider } from '__tests__/util/graphqlMocking';
+import { render, waitFor } from '__tests__/util/testingLibraryReactMock';
+import theme from '../../../theme';
+import FixCommitmentInfo from './FixCommitmentInfo';
+import { mockInvalidStatusesResponse } from './FixCommitmentInfoMocks';
+import { GetInvalidStatusesQuery } from './GetInvalidStatuses.generated';
+
+jest.setTimeout(8000);
+
+const mockEnqueue = jest.fn();
+jest.mock('notistack', () => ({
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  ...jest.requireActual('notistack'),
+  useSnackbar: () => {
+    return {
+      enqueueSnackbar: mockEnqueue,
+    };
+  },
+}));
+
+const accountListId = 'test121';
+const router = {
+  pathname: '/accountLists/[accountListId]/tools/fixCommitmentInfo',
+  query: { accountListId: accountListId },
+  push: jest.fn(),
+};
+
+const setContactFocus = jest.fn();
+
+const Components = ({
+  mockNodes = mockInvalidStatusesResponse,
+}: {
+  mockNodes?: ErgonoMockShape[];
+}) => (
+  <SnackbarProvider>
+    <ThemeProvider theme={theme}>
+      <TestRouter router={router}>
+        <TestWrapper>
+          <GqlMockedProvider<{
+            GetInvalidStatuses: GetInvalidStatusesQuery;
+          }>
+            mocks={{
+              GetInvalidStatuses: {
+                contacts: {
+                  nodes: mockNodes,
+                },
+              },
+            }}
+          >
+            <VirtuosoMockContext.Provider
+              value={{ viewportHeight: 1000, itemHeight: 100 }}
+            >
+              <FixCommitmentInfo
+                accountListId={accountListId}
+                setContactFocus={setContactFocus}
+              />
+            </VirtuosoMockContext.Provider>
+          </GqlMockedProvider>
+        </TestWrapper>
+      </TestRouter>
+    </ThemeProvider>
+  </SnackbarProvider>
+);
+
+describe('FixCommitmentContact', () => {
+  it('default with test data', async () => {
+    const { getByText, debug } = render(<Components />);
+    debug(undefined, Infinity);
+    await waitFor(() => {
+      expect(getByText('Fix Commitment Info')).toBeInTheDocument();
+      expect(
+        getByText('You have 2 partner statuses to confirm.'),
+      ).toBeInTheDocument();
+    });
+  });
+
+  it('has correct styles', async () => {
+    const { getByTestId, debug } = render(<Components />);
+    debug(undefined, Infinity);
+    const home = getByTestId('Home');
+
+    expect(home.classList.contains('css-1ruq7cl-outer')).toBe(true);
```

Comment:
Since MUI creates the HTML, I'm unsure how long this class will be this untill it's changed. It is properly better off to add your own class.
```suggestion
    expect(home.classList.contains('css-1ruq7cl-outer')).toBe(true);
```